src/components/pages/TeachersPage/SendMessages.css.tsx

```diff
@@ -0,0 +1,28 @@
+import { css } from '@emotion/react';
+
+const teacherName = css`
+  text-align: center;
+  color: purple;
+  font-weight: bold;
+  font-size: 18px;
+`;
+
+// TODO: This "message" css is identical to the one in StudentsPage/SendMessages.css.  It can be refactored to be only defined once.
+const message = css`
+  border-radius: 20px;
+  border: 2px solid lightgrey;
```

Comment:
Nit: In all honesty we could refactor all ChatBox Conversations to be more generic.  We have a lot of duplicated css and component logic that could be refactored and shared between the students and teachers page components.

The new SendMessage Component in the teachers page is basically copy and paste from the students SendMessages component with the "Peer Typing/Left" and "Teacher Ended" Logic Stripped out and a small UI tweak.

![image](https://user-images.githubusercontent.com/2927894/219068472-b7c498bb-6ad5-41b1-a353-96faa2f6cbcd.png)

Reply:
I'm happy you put words to this! I'd been thinking the same about how there is so much duplicate code between the TeachersPage chatbot and the StudentsPage chatbot.    I think tackling this duplicated code is a separate issue from this PR.  I'll start a discussion about reducing duplication in our Slack and will include Karl.